src/Html.php

```diff
@@ -1218,7 +1218,9 @@ public static function includeHeader(
 
         $tpl_vars['js_files'][] = ['path' => 'lib/base.js'];
         $tpl_vars['js_files'][] = ['path' => 'js/webkit_fix.js'];
-        $tpl_vars['js_modules'][] = ['path' => 'build/vue/app.js'];
+        // Vue entrypoint needs no version param because the chunks already have the hash in the name
+        // The entrypoint itself should rarely change but devs could also always use HMR
+        $tpl_vars['js_modules'][] = ['path' => 'build/vue/app.js', 'options' => ['no_version' => true]];
```

Comment:
We need a `version` parameter at least in production environments, to be sure that the file from a previous version is never used when extensive cache headers are forced by web server rules.

Reply:
I handled cache busting differently here. Vite now puts the hash in the file name of the "app.js" as well, and GLPI simply refers to the manifest to determine the path to use.

https://vite.dev/guide/backend-integration